src/Joblify.Core.Data/Context/JoblifyDbContext.cs

```diff
@@ -5,11 +5,23 @@ namespace Joblify.Core.Data.Context
 {
     public class JoblifyDbContext : DbContext
     {
+        public JoblifyDbContext()
+        {
+        }
+
         public JoblifyDbContext(DbContextOptions<JoblifyDbContext> options) : base(options) { }
 
         public virtual DbSet<User> Users { get; set; }
         public virtual DbSet<ExternalProvider> ExternalProviders { get; set; }
         public virtual DbSet<Role> Roles { get; set; }
         public virtual DbSet<File> Files { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+                optionsBuilder.UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=Joblify;Trusted_Connection=True;");
```

Comment:
connection string shouldn't be hardcoded

Reply:
So I should add connectionstring to appsettings.json?

Reply:
yes, then use it to instantiate context in startup class